daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/LedgerInteraction.scala

```diff
@@ -463,6 +476,19 @@ class IdeClient(val compiledPackages: CompiledPackages) extends ScriptLedgerClie
     Future.fromTry(result)
   }
 
+  override def submitMustFail(party: SParty, commands: List[ScriptLedgerClient.Command])(
+      implicit ec: ExecutionContext,
+      mat: Materializer): Future[Either[Unit, Unit]] = {
+    submit(party, commands)
+      .map({
+        case Right(_) => Left(())
+        case Left(_) => Right(())
```

Comment:
We expect to never hit this right? Because in the script service you will always run against the current script library. If so, it might be useful to add a comment.

Reply:
Yes, it's only listed for completeness. I've added a comment.